Gemfile

```diff
@@ -48,8 +46,6 @@ gem 'will_paginate'
 # Continuation of CanCan (authoriation Gem for RoR)
 gem 'cancancan'
 
-# See https://github.com/rails/execjs#readme for more supported runtimes
-gem 'therubyracer', platforms: :ruby
```

Comment:
I am not sure either. However, as I dropped the `less-rails` dependency with [this PR](https://github.com/openHPI/codeharbor/pull/338), I think we can now use [these setup instructions for twitter-bootstrap-rails](https://github.com/seyhunak/twitter-bootstrap-rails/#installing-the-css-stylesheets). There, the authors mention that only the one gem instead of the three gems (incl. `less-rails` and one of the `execJS` runtimes) are needed.

Reply:
sounds good